test/check-application

```diff
@@ -27,37 +27,48 @@ class TestApplication(testlib.MachineCase):
 
         # show image listing toggle
         b.wait_present('#containers-images tr:contains("busybox:latest")')
-        b.wait_visible('#containers-images tr:contains("busybox:latest")')
         b.click('#containers-images tbody tr:contains("busybox:latest") td.listing-ct-toggle')
         b.wait_present('#containers-images tbody tr:contains("busybox:latest") + tr button.btn-delete')
 
         # run a container (will exit immediately)
-        m.execute("podman run -d --name swamped-crate busybox sh")
+        m.execute("podman run -d --name test-sh alpine sh")
+        # run a container
+        m.execute("podman run -d --name swamped-crate busybox sleep 1000")
 
         # FIXME: UI needs to listen to change signals and update automatically
+
         b.reload()
-        b.wait_present("#content")
         b.enter_page("/podman")
         b.wait_present("#containers-containers-filter button")
 
-        # show all containers to interact with stopped ones
+        # show all containers and check status
         b.click("#containers-containers-filter button")
-        b.wait_visible("#containers-containers-filter .dropdown-menu")
+        b.wait_present("#containers-containers-filter .dropdown-menu")
         b.click("#containers-containers-filter li[data-data='all'] a")
         b.wait_present("#containers-containers")
+        # check containers name
+        b.wait_present('#containers-containers tr:contains("swamped-crate")')
         b.wait_present('#containers-containers tr:contains("busybox:latest")')
-        b.wait_visible('#containers-containers tr:contains("busybox:latest")')
-
+        b.wait_present('#containers-containers tr:contains("test-sh")')
+        b.wait_present('#containers-containers tr:contains("alpine:latest")')
```

Comment:
This is admittedly a bit picky, but this doesn't assert that the pod names match the image names, i. e. they are in the line where they belong to. A common pattern is to give the `<tr>` some unique attribute, like `data-name="swamped-crate"`, so that you can first select the row and then assert that it contains the container name, command, etc. E. g. 

     b.wait_in_text('#containers-containers tr[data-name="swamped-crate"]', 'swamped-crate')
     b.wait_in_text('#containers-containers tr[data-name="swamped-crate"]', 'busybox:latest')
     b.wait_in_text('#containers-containers tr[data-name="swamped-crate"]', 'running')

An even stricter way is to check the n-th row, like

     b.wait_in_text('#containers-containers tr:nth-of-type(1)', 'swamped-crate')

This of course requires that the order of the rows is predictable. This is usually a good idea anyway for UIs, i. e. usually one sorts them alphabetically by name, or first by image name then by container name, or by age or something. However, the current code doesn't do this (would be good to add, though, and it's trivial to do), so you can't use this variant yet.

Your variant below with `tr:contains()` is also very interesting, and avoids the need for data-* attributes. So I suggest to consistently use that in the above checks as well.

Also: It's also  usually easier to read and maintain to check all properties of one container (like in my example above), then the next one container, i. e. row-by-row. The above checks "column by column", and IMHO that's harder to follow mentally. (But that's a matter of taste, of course).